### ISSUE TITLE
Bar line can not be altered

### ISSUE THREAD

**Issue by VictorEijkhout** [musescore/MuseScore#19422]
### Issue type

Engraving bug

### Bug description

This is a score from the experimental pdf importer. Most errors I was able to correct, but there is a double bar line that I can not edit away. It doesn't seem to be of any type in the "barlines" palette. See attached score.

### Steps to reproduce

1. Sorry, no idea. Please look at the innards of the attached file. There is some strange corruption.
[bar.zip](https://github.com/musescore/MuseScore/files/12641484/bar.zip)

### MuseScore Version

4.1.1

### Regression

I don't know

### Operating system

MacOS 13

**Comment by MarcSabatella** [comment]
Under "steps to reproduce", what is needed are the steps to reproduce the problem given the file you attached.  I tried selecting some of the barlines and they deleted as expected.   It does appear they were added individally per staff, so they need to be deleted one at a time, but aside from that it seems to wrk as expected for me.

In any case, the PDF converter is driven by the open source softrware Audiveris, so any errors in how it performs a conversion should be reported to ther folks who maintain that.  I guess that is probably https://github.com/Audiveris/audiveris, unless we're still using the older version at https://github.com/Audiveris/audiveris-eg.
